### ISSUE TITLE
Segfault with getcone

### ISSUE THREAD

**Issue by blegat** [JuliaOpt/Mosek.jl#186]
```julia
julia> using Mosek

julia> Mosek.getversion()
(9, 0, 97)

julia> versioninfo()
Julia Version 1.2.0
Commit c6da87ff4b (2019-08-20 00:03 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)

julia> task = maketask()
Empty Task


julia> getcone(task, 1)

signal (11): Segmentation fault
in expression starting at REPL[6]:1
MSK_getcone at /home/blegat/mosek/9.0/tools/platform/linux64x86/bin/libmosek64.so.9.0 (unknown line)
macro expansion at /home/blegat/.julia/packages/Mosek/43Z2H/src/Mosek.jl:30 [inlined]
#136 at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:2353 [inlined]
disable_sigint at ./c.jl:446
getconeinfo at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:2352
getcone at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:2320 [inlined]
getcone at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:2315
unknown function (ip: 0x7f96ac11d097)
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2197
do_call at /buildworker/worker/package_linux64/build/src/interpreter.c:323
eval_value at /buildworker/worker/package_linux64/build/src/interpreter.c:411
eval_stmt_value at /buildworker/worker/package_linux64/build/src/interpreter.c:362 [inlined]
eval_body at /buildworker/worker/package_linux64/build/src/interpreter.c:772
jl_interpret_toplevel_thunk_callback at /buildworker/worker/package_linux64/build/src/interpreter.c:884
unknown function (ip: 0xfffffffffffffffe)
unknown function (ip: 0x7f96c3721e8f)
unknown function (ip: 0xffffffffffffffff)
jl_interpret_toplevel_thunk at /buildworker/worker/package_linux64/build/src/interpreter.c:893
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:815
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:764
jl_toplevel_eval_in at /buildworker/worker/package_linux64/build/src/toplevel.c:844
eval at ./boot.jl:330
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2191
eval_user_input at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:86
macro expansion at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:118 [inlined]
#26 at ./task.jl:268
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2191
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1614 [inlined]
start_task at /buildworker/worker/package_linux64/build/src/task.c:596
unknown function (ip: 0xffffffffffffffff)
Allocations: 9806515 (Pool: 9804370; Big: 2145); GC: 20
[1]    11542 segmentation fault (core dumped)  julia --color=yes --project
```
This also segfault but isn't it unexpected that the `getcone(task, 2)` does not error like `getcone(task, 3)` does ?
```julia
julia> using Mosek

julia> task = maketask()
Empty Task


julia> appendvars(task, 3)

julia> appendcone(task, MSK_CT_RQUAD, 0.0, Cint[1, 2, 3])

julia> getcone(task, 1)
(Mosek.MSK_CT_RQUAD, 0.0, 3, Int32[1, 2, 3])

julia> getcone(task, 2)
(Mosek.MSK_CT_QUAD, 0.0, 0, Int32[])

julia> getcone(task, 3)
ERROR: MosekError(1204, "The index value 2 occurring in argument 'k' is too large.")
Stacktrace:
 [1] getconeinfo(::Mosek.Task, ::Int32) at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:2357
 [2] getcone at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:2320 [inlined]
 [3] getcone(::Mosek.Task, ::Int64) at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:2315
 [4] top-level scope at REPL[8]:1
```

**Comment by blegat** [comment]
Note that I also get a segfault with `removecones`:
```julia
julia> task = maketask()
Empty Task


julia> removecones(task, [0])

signal (11): Segmentation fault
in expression starting at REPL[3]:1
i32vec_fills32 at /home/blegat/mosek/9.0/tools/platform/linux64x86/bin/libmosek64.so.9.0 (unknown line)
MSK_removecones at /home/blegat/mosek/9.0/tools/platform/linux64x86/bin/libmosek64.so.9.0 (unknown line)
macro expansion at /home/blegat/.julia/packages/Mosek/43Z2H/src/Mosek.jl:30 [inlined]
#572 at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:8387 [inlined]
disable_sigint at ./c.jl:446 [inlined]
removecones at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:8386
removecones at /home/blegat/.julia/packages/Mosek/43Z2H/src/msk_functions.jl:8383
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2197
do_call at /buildworker/worker/package_linux64/build/src/interpreter.c:323
eval_value at /buildworker/worker/package_linux64/build/src/interpreter.c:411
eval_stmt_value at /buildworker/worker/package_linux64/build/src/interpreter.c:362 [inlined]
eval_body at /buildworker/worker/package_linux64/build/src/interpreter.c:772
jl_interpret_toplevel_thunk_callback at /buildworker/worker/package_linux64/build/src/interpreter.c:884
unknown function (ip: 0xfffffffffffffffe)
unknown function (ip: 0x7ff5c7c9d80f)
unknown function (ip: (nil))
jl_interpret_toplevel_thunk at /buildworker/worker/package_linux64/build/src/interpreter.c:893
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:815
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:764
jl_toplevel_eval_in at /buildworker/worker/package_linux64/build/src/toplevel.c:844
eval at ./boot.jl:330
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2191
eval_user_input at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:86
macro expansion at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.2/REPL/src/REPL.jl:118 [inlined]
#26 at ./task.jl:268
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2191
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1614 [inlined]
start_task at /buildworker/worker/package_linux64/build/src/task.c:596
unknown function (ip: 0xffffffffffffffff)
Allocations: 5524824 (Pool: 5523783; Big: 1041); GC: 11
[1]    16179 segmentation fault (core dumped)  julia --color=yes
```

**Comment by ulfworsoe** [comment]
I certainly hope that any segfault is unexpected. It looks like an MOSEK library error and not an API error. I'll take a look.

**Comment by ulfworsoe** [comment]
This was a MOSEK solver issue. It has been fixed in the latest 9.1 build.
